modules/ROOT/pages/sync-with-couchbase-server.adoc

```diff
@@ -129,16 +129,16 @@ As stated, mobile metadata is not kept in the document, but in a system extended
 The N1QL query language {fnref-cbs5x0} supports the ability to query these extended attributes (XATTRS) and hence the document's sync metadata -- see: <<simple-query>>.
 
 [#simple-query]
-.Querying XATTRS-bsed sync metadata
+.Querying XATTRS-based sync metadata
 ====
 [source,sql]
 ----
 SELECT meta().xattrs._sync FROM scope.collection WHERE meta().id = "mydocId"
 ----
 ====
 
-WARNING: The sync metadata is maintained internally by {sgw} and its structure can change at any time.
-It should not be used to drive business logic of applications. The direct use of the N1QL query is *unsupported* and must not be used in production environments.
+WARNING: {sgw} maintains the sync metadata internally, and its structure can change at any time. Applications must not use it for business logic. We do not support directly using N1QL queries or modifying the internal sync metadata contents in production environments. +
```

Comment:
I'm a little bit unclear. 

We do not support using N1QL queries to modify the internal sync metadata contents in production environments?

Reply:
We do not support directly using N1QL or modifying internal sync metadat for business logic, also do not support using in production environments.
Pushed a commit on this.